gcp/kbn-dev.sh

```diff
@@ -7,21 +7,42 @@ TYPE=$2
 VALUE=$3
 
 REPO_URL=${REPO_URL:-"https://github.com/elastic/kibana"}
+
+GCP_NAME_PREFIX=${KBN_GCP_PREFIX:-"kbn-dev-v1"}
```

Comment:
now the usual prefix is configureable so you can 

`export KBN_GCP_PREFIX=my-nice-gcp-name-prefix`

to use a different prefix in GCP